SeaFightLogic/Ships/Interfaces/IHealing.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;

```

Comment:
Any code in this file that would apply these namespaces. Always delete useless namespaces

Reply:
done here